tests/api_tests/nodes/test_views.py

```diff
@@ -951,6 +951,103 @@ def test_node_children_list_does_not_include_unauthorized_projects(self):
         assert_equal(len(res.json['data']), 1)
 
 
+class TestNodeChildCreate(ApiTestCase):
+
+    def setUp(self):
+        super(TestNodeChildCreate, self).setUp()
+        self.user = UserFactory.build()
+        self.user.set_password('justapoorboy')
+        self.user.save()
+        self.basic_auth = (self.user.username, 'justapoorboy')
+
+        self.project = ProjectFactory(creator=self.user, is_publc=True)
+
+        self.url = '/{}nodes/{}/children/'.format(API_BASE, self.project._id)
+
+        self.title = 'Cool Project'
+        self.description = 'A Properly Cool Project'
+        self.category = 'data'
+
+        self.user_two = UserFactory.build()
+        self.user_two.set_password('justapoorboy')
```

Comment:
We have just learned that there's a better way to do this. Instead of UserFactory, use UserAuthFactory, then you won't need to set an auth variable nor set the username and password.